static-site/app/groups/[[...groups]]/group-listing-page.tsx

```diff
@@ -155,13 +132,11 @@ export default function GroupListingPage(): React.ReactElement {
         <h2 className="centered">Available Narratives</h2>
       </ScrollableAnchor>
 
-      {dataLoaded && (
-        <DatasetSelect
-          datasets={narratives}
-          columns={datasetColumns}
-          title={"Filter Narratives"}
-        />
-      )}
+      <ListResources
```

Comment:
_non-blocking_

In brief testing, both instances of `ListResources` share the filter queries set via URL params, so filtering datasets will also filter narratives and vice versa. I could see that as a feature, but it's kind of weird since there are two search inputs controlled by the same URL param.